index.js

```diff
@@ -1,12 +1,15 @@
 /* eslint-env node */
 'use strict';
 
+const transform = require('./transforms/input');
+
 module.exports = {
   name: 'ember-2-legacy',
 
   included() {
     this._super.included.apply(this, arguments);
 
+    this.import('vendor/ember/ember-template-compiler.js');
```

Comment:
@rwjblue Is there a better way to do this only for the addons tests? or should I just include it in tests/index.html directly?

Reply:
Hmm, why do you need it?

Reply:
I think the thing you will need to do is modify the tests to `import hbs from 'htmlbars-inline-precompile'`. If you do that, you will see the deprecations on the console at build time but you cannot do `assert.expectDeprecation`.  In other words, you can only confirm that the old functionality works, but not that it deprecates.  IMHO, thats a fine trade off, what do you think?

Reply:
I'm fine with not testing the deprecations as that was the only thing keeping me off of `hbs`. I was also worried about all of the logging in the console and not wanting that to show up after someone installs the addon but I just tested and that wont happen. I guess its because the tests are included locally but not when included in an app.